web/static/js/components/idea_submission_form.jsx

```diff
@@ -49,7 +49,7 @@ export class IdeaSubmissionForm extends Component {
     const { currentUser } = this.props
     event.preventDefault()
     const newIdea = { ...this.state, userId: currentUser.id }
-    if (!newIdea.assigneeId) { newIdea.assigneeId = null }
```

Comment:
rather than doing this assignment just before submission, we can just set assigneeId to `null` where we declare `this.state = { ... }` in the constructor

Reply:
Done